packages/expo-sqlite/src/SQLite.types.ts

```diff
@@ -0,0 +1,223 @@
+// Type definitions for websql
+// Project: http://www.w3.org/TR/webdatabase/
+// Definitions by: TeamworkGuy2 <https://github.com/TeamworkGuy2>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// W3C spec: http://www.w3.org/TR/webdatabase/#database
+// Spec revision: 2010-11-18
+// NOTE: the W3C websql spec has been deprecated
+
+//[Callback = FunctionOnly, NoInterfaceObject]
+export interface DatabaseCallback {
+  /*handleEvent*/ (database: Database): void;
+}
+
+/** 4.3 Asynchronous database API - The transaction() and readTransaction() methods takes
+ * one to three arguments. When called, these methods must immediately return and then
+ * asynchronously run the transaction steps with the transaction callback being the
+ * first argument, the error callback being the second argument, if any, the success
+ * callback being the third argument, if any, and with no preflight operation or
+ * postflight operation
+ */
```

Comment:
It's taken directly from https://www.w3.org/TR/webdatabase/ and I would rather have some comment from https://docs.expo.io